cypress/integration/dashboard_spec.js

```diff
@@ -0,0 +1,17 @@
+describe('Details', () => {
+  beforeEach(() => {
+    cy.load()
+  });
+  it('Should show all movies', () => {
+    cy.get('.card').should('be.visible')
+    cy.get('.article').should('not.exist')
+  })
+
+  it('Should switch to movie details view', () => {
+    cy.get('.card').click()
+      .get('.card').should('not.exist')
+      .get('.details').should('be.visible')
+      .get('.details').contains('Money Plane')
+  })
+
+});
```

Comment:
This is a great foundation for implementing router. We can easily refactor when we do. As for now, it tests that are page is displaying what it should be after a movie card is clicked on.